app/models/alchemy/content/factory.rb

```diff
@@ -19,7 +19,8 @@ def build(element, essence_hash)
         if (description = content_description(element, essence_hash)).blank?
           raise ContentDefinitionError, "No description found in elements.yml for #{essence_hash.inspect} and #{element.inspect}"
         else
-          new(name: description['name'], element_id: element.id)
+          translate = description['translate'] == false ? false : true
+          new(name: description['name'], element_id: element.id, translate: translate)
```

Comment:
migration is in: https://github.com/LootCrate/lootcrate/pull/3262

I did place a default of true in the db and wrote a rake task to add true to the current contents.  

the way I wrote it should always default to true unless we specifically set `translate: false` in `yml`.

I'm up for changing it if we think this way is cleaner.